styles.css

```diff
@@ -0,0 +1,8 @@
+body {
+  font-family: 'Roboto', sans-serif;
+}
+
+.section__title, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/html_2.0/pullRequest?prid=6290605)